app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -52,6 +52,7 @@ protected function validator(array $data)
         return Validator::make($data, [
             'name' => ['required', 'string', 'max:255'],
             'email' => ['required', 'string', 'email', 'max:255', 'unique:users'],
+            'role'=> ['required','string'],
```

Comment:
Since there are only specific roles, ensure that user can only enter those roles, not anything else. Tip. Use the `Rule::in('value1', 'value2')`

Reply:
I'm failing to find where to use this.

Reply:
Check this out: https://laravel.com/docs/master/validation#rule-in